client/signup/steps/secure-your-brand/index.jsx

```diff
@@ -0,0 +1,220 @@
+		};
+
+		this.props.submitSignupStep( step, { cartItems: null } );
+		this.props.goToNextStep();
+	}
+
+	getFormattedDomain( domain ) {
+		const domainBase = domain.substring( 0, domain.indexOf( '.' ) );
+		const domainTld = domain.substring( domain.indexOf( '.' ) );
+		return (
+			<>
+				{ domainBase }
+				<span>{ domainTld }</span>
+			</>
+		);
+	}
+
+	recommendedDomains() {
+		const { translate, secureYourBrand } = this.props;
+		const domain = this.getDomainName();
+		this.getDomainName();
+		const productData = secureYourBrand.product_data;
+		const currency = secureYourBrand.currency;
+		const discountedCost = formatCurrency( secureYourBrand.discounted_cost, currency, {
+			stripZeros: true,
+		} );
+		const totalCost = formatCurrency( secureYourBrand.total_cost, currency, { stripZeros: true } );
+		const classNames = {
+			'is-primary': true,
+		};
+
+		return (
+			<div className="secure-your-brand">
+				<QuerySecureYourBrand domain={ domain } />
+				<Card>
+					<div className="secure-your-brand__available">
+						<Gridicon icon="checkmark-circle" />
+						<span>{ translate( '%(domain)s is available', { args: { domain } } ) }</span>
+					</div>
+					<CardHeading size={ 20 } tagName="h3">
+						{ translate( 'Domain signup bundle' ) }
+					</CardHeading>
+					<div className="secure-your-brand__domains">
+						{ productData?.map( ( suggestion ) => (
+							<div className="secure-your-brand__domain" key={ suggestion.domain }>
+								<div className="secure-your-brand__domain-name">
+									{ this.getFormattedDomain( suggestion.domain ) }
+								</div>
+								<div className="secure-your-brand__cost">
+									{ translate( '%(price)s/year', {
+										args: {
+											price: formatCurrency( suggestion.cost, currency, { stripZeros: true } ),
+										},
+									} ) }
+								</div>
+							</div>
+						) ) }
+					</div>
+					<div className="secure-your-brand__total">
+						<div className="secure-your-brand__total-label">{ translate( 'Total' ) }</div>
+						<div className="secure-your-brand__total-description">
+							<span className="secure-your-brand__discount-label">
+								{ translate( '%(discountedCost)s for your first year', {
+									args: { discountedCost },
+								} ) }
+							</span>
+							<span className="secure-your-brand__total-cost">
+								{ translate( '%(totalCost)s/year', {
+									args: { totalCost },
+								} ) }
+							</span>
+						</div>
+					</div>
+					<CardHeading size={ 16 } tagName="h4">
+						{ translate(
+							'For just {{del}}%(totalCost)s{{/del}} %(discountedCost)s for the first year, you will:',
+							{
+								args: {
+									discountedCost,
+									totalCost,
+								},
+								components: {
+									del: <del />,
+								},
+							}
+						) }
+					</CardHeading>
+					<div className="secure-your-brand__benefits">
+						<ul>
+							<li>{ translate( 'Prevent anyone from registering (and misusing) your name' ) }</li>
```

Comment:
Since this is an `en` only A/B test with unknown outcome, we don't need to submit strings for translation. That could save a few $$ and human effort :). 
We could remove the  `translate()` calls, but since that would be a bit of effort, it's also fine to keep them and  after deploying the PR, we coordinate with Team Global to deprioritize these strings so that they don't get submitted into the weekly kit.